### DISCUSSION TITLE
[Esame 15 gennaio 2024] Esercizio 1

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#450] [Esami M-Z]
![15 gen es.1](https://github.com/user-attachments/assets/dbea0a60-a2c7-4ecb-834b-5cf047050dd7)

<details>
  <summary>soluzione del prof</summary>
  <br>
  <img src="https://github.com/user-attachments/assets/4444f67d-ed7b-43ca-8efa-ce67ae5e728f" alt="soluzione"  />
  <img src="https://github.com/user-attachments/assets/499a90e3-0b31-4cab-baaf-ba987e5d0d35" alt="soluzione 2"  />
  
</details>

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/ac6c732d-d8dc-4b96-a65d-f22c77e46a87)

**Comment by niccolo-fato** [comment]
<img width="979" height="600" alt="image" src="https://github.com/user-attachments/assets/b67f7b37-0f35-4055-8e0f-6a78c8c620e1" />
